Solutions/Menes.Hosting.AspNetCore/Menes/Internal/HttpRequestParameterBuilder.cs

```diff
@@ -422,6 +527,9 @@ private async Task<object> ConvertBodyAsync(OpenApiSchema schema, Stream body)
                 return true;
             }
 
+            //// No default value handling since Path parameters aren't optional if they've been defined (and therefore default values are redundant).
+            //// This is as per the the Open API spec, as seen here: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#parameterObject.
+
```

Comment:
Quadruple backslashes are generally used to temporarily comment out code without Roslynator complaining that you're not adhering to the rules around comments. Genuine comments like this should use double backslashes.

Reply:
Fixed.